.github/actions/setup-rust/action.yaml

```
@@ -0,0 +1,12 @@
+name: 'Setup rust'
+description: 'Setup rust environment for databend'
+runs:
+  using: "composite"
+  steps:
+    - name: Install Protoc
+      uses: arduino/setup-protoc@v1
```

The action queries the GitHub API to fetch releases data, to avoid rate limiting, pass the default token with the repo-token variable

```suggestion
    - name: Install Protoc
      uses: arduino/setup-protoc@v1
      with:
        repo-token: ${{ secrets.GITHUB_TOKEN }}
```

---

> API rate limit exceeded for 13.105.49.52. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)

---

thank you for reminding, i've changed as you suggested